client/app/telecom/dashboard/translations/Messages_fr_FR.xml

```diff
@@ -1,48 +1,55 @@
+
+    <translation id="telecom_dashboard_task_title" qtlid="305005">Mes tâches</translation>
+
+    <translation id="telecom_dashboard_FAQ_title" qtlid="300370">Guides</translation>
+    <translation id="telecom_dashboard_FAQ_Activate" qtlid="305018">Activation de mes services</translation>
+    <translation id="telecom_dashboard_FAQ_managed_modem" qtlid="305031">Configurer mon modem</translation>
+    <translation id="telecom_dashboard_FAQ_init_modem" qtlid="305044">Réinitialiser mon modem</translation>
+    <translation id="telecom_dashboard_FAQ_restore_service">Rétablir son service</translation>
+    <translation id="telecom_dashboard_incident_impact" qtlid="305057">Incident détecté</translation>
+
+    <translation id="telecom_dashboard_services_title" qtlid="305070">Mes Services</translation>
+
+    <translation id="telecom_dashboard_services_internet" qtlid="246975">Internet</translation>
+    <translation id="telecom_dashboard_services_telephony" qtlid="29299">Téléphonie</translation>
+    <translation id="telecom_dashboard_services_sms" qtlid="28243">SMS</translation>
+    <translation id="telecom_dashboard_services_hubic" qtlid="305083">HubiC</translation>
+    <translation id="telecom_dashboard_services_exchange" qtlid="80430">Exchange</translation>
+    <translation id="telecom_dashboard_services_fax" qtlid="32926">Fax</translation>
+
+    <translation id="telecom_dashboard_services_internet_none" qtlid="305096">Aucune ligne xDSL</translation>
+    <translation id="telecom_dashboard_services_telephony_none" qtlid="305109">Pas de téléphonie</translation>
+    <translation id="telecom_dashboard_services_sms_none" qtlid="305122">Pas de SMS</translation>
+    <translation id="telecom_dashboard_services_hubic_none" qtlid="305135">Pas d'HubiC</translation>
+    <translation id="telecom_dashboard_services_exchange_none" qtlid="305148">Pas d'Exchange</translation>
+    <translation id="telecom_dashboard_services_fax_none" qtlid="305161">Pas de fax</translation>
+
+    <translation id="telecom_dashboard_services_incidents_one" qtlid="305174">incident détecté</translation>
+    <translation id="telecom_dashboard_services_incidents_other" qtlid="305187">incidents détectés</translation>
+
+    <translation id="telecom_dashboard_auth_failed" qtlid="305200">Impossible de récupérer les informations de
+        l'utilisateur.
+    </translation>
+    <translation id="telecom_dashboard_express_lite_desc" qtlid="455712">Express Lite comprend donc un modem offrant un
+        accès à internet et une ligne VoIP pour <span class="ad-price font-weight-bold fs14">22,99€ HT</span>&#160;/&#160;mois.
+    </translation>
+    <translation id="telecom_dashboard_express_lite_title" qtlid="304888">OVH lance une nouvelle offre de connexion ADSL
+        et VDSL incluant l'essentiel.
+    </translation>
+    <translation id="telecom_dashboard_discover" qtlid="304927">Découvrir l'offre</translation>
```

Comment:
Did you change something apart from the indentation? It's harder to notice :/
Maybe we should revert this change and handle the indentation in another fix.